### PR TITLE
TP4 documentation and release notes updates

### PR DESCRIPTION
**Pull request by geekspertise**
I updated the service mesh documentation and release notes for TP4. @tvieira @knrc and @JStickler can you please review?

### CODE REVIEW

File: servicemesh-release-notes/topics/rn-fixed-issues.adoc [hunk 1]
```diff
@@ -8,14 +8,18 @@ Fix - What did we change to fix the problem?
...
 
 https://issues.jboss.org/browse/MAISTRA-13[MAISTRA-13] - Installer should determine release version from installed components.  At present, the installer queries the yaml file, however if the yaml has been modified, the installer is not able to remove an older version.
 
+https://issues.jboss.org/browse/MAISTRA-79[MAISTRA-79] - Running the `istiooc cluster up` command results in the istio-operator namespace deploying a pod responsible for continually ensuring the Elastisearch sysctl requirements are met. This loop runs constantly causing a significant load on the system running the cluster.
+
+https://github.com/Maistra/openshift-ansible/pull/19/[MASITRA-21] -  The default in the installer of 512Mi was too low for tracing.  Increased default Elasticsearch memory from 512 MB to 1 GB.
```
Comment (knrc) [inline servicemesh-release-notes/topics/rn-fixed-issues.adoc]: MAISTRA-21 and not MASITRA-21.  This seems to have been an old misspelling copied down to here.

File: servicemesh-release-notes/topics/rn-known-issues.adoc [hunk 2]
```diff
@@ -28,12 +28,12 @@ https://github.com/istio/old_issues_repo/issues/115
 ////
 * The istio-init container requires a privileged security context or at least to run as root and to have the NET_ADMIN capability.  The istio-init container needs to be privileged because it needs to properly configure the iptables rules in the pod in order to intercept network connections.  The team is currently investigating ideas for ways to reduce the privileges required by Istio.
 
+https://issues.jboss.org/browse/MAISTRA-61[MAISTRA-61] After all applicable resources are deployed to OpenShift, Istio takes several minutes to begin routing to the correct pod.
+
```
Comment (knrc) [inline servicemesh-release-notes/topics/rn-known-issues.adoc]: Could we change this description to be something like the following?

> After all applicable resources are deployed to OpenShift, Istio sidecars may lose information about their routes and can no longer communicate with services until the next update is received

**Review by knrc — CHANGES_REQUESTED** [review]

File: servicemesh-release-notes/topics/rn-fixed-issues.adoc (hunk 1 above)
Comment (geekspertise) [reply servicemesh-release-notes/topics/rn-fixed-issues.adoc]: @knrc I fixed this.

File: servicemesh-release-notes/topics/rn-known-issues.adoc (hunk 2 above)
Comment (geekspertise) [reply servicemesh-release-notes/topics/rn-known-issues.adoc]: @knrc I adopted your wording.

**Comment by JStickler** [comment]
LGTM

**Comment by tvieira** [comment]
LGTM

File: servicemesh-release-notes/topics/rn-new-features.adoc [hunk 3]
```diff
@@ -5,6 +5,10 @@ Feature – Describe the new functionality available to the customer.  For enhan
 Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.).   For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.  There may not have been a 'problem' previously, but system behaviour may have changed.
 Result – If changed, describe the current user experience
 ////
+= New features Technology Preview 4
+
+The release of {ProductName} adds support for Kiali 0.9.x.
```
Comment (brian-avery) [inline servicemesh-release-notes/topics/rn-new-features.adoc]: Might make sense here to mention that tp4 is based on Istio 1.0.3

**Review by brian-avery — APPROVED** [review]

File: servicemesh-release-notes/topics/rn-new-features.adoc (hunk 3 above)
Comment (geekspertise) [reply servicemesh-release-notes/topics/rn-new-features.adoc]: @brian-avery I've made this change.

**Comment by geekspertise** [comment]
@kalexand-rh @vikram-redhat I've made all of the updates for Service Mesh TP4, handled comments from review, and this is now ready to merge.

**Comment by kalexand-rh** [comment]
@geekspertise, is TP4 going live today?

**Comment by geekspertise** [comment]
> @geekspertise, is TP4 going live today?

Yes. I spoke with @knrc and we are good to go.

**Comment by kalexand-rh** [comment]
Ok. Merging and CP'ing to 3.11.

**Pull request merged**